init_arrays not found in v6.0-rc

I'm building with `cmake -Bbuild -H. -DMODEL=cuda -DCUDA_ARCH=sm_90 -DCMAKE_CUDA_COMPILER=nvcc` using gcc 11.5.0 and nvcc 13.2.0. 

In `v6.0-rc` I get this error when building.
```
[ 33%] Building CUDA object CMakeFiles/cuda-stream.dir/src/cuda/CUDAStream.cu.o
[ 66%] Building CXX object CMakeFiles/cuda-stream.dir/src/main.cpp.o
/home/almalinux/BabelStream/src/main.cpp: In instantiation of ‘void run() [with T = float]’:
/home/almalinux/BabelStream/src/main.cpp:97:15:   required from here
/home/almalinux/BabelStream/src/main.cpp:280:42: error: ‘class Stream<float>’ has no member named ‘init_arrays’; did you mean ‘get_arrays’?
  280 |   auto initElapsedS = time([&] { stream->init_arrays(startA, startB, startC); });
      |                                  ~~~~~~~~^~~~~~~~~~~
      |                                  get_arrays
```
In `v5.0` it builds successfully.

I experienced almost same issue. My Back-End case was OpenCL.

```
$ cmake --build build-ocl -j
[ 33%] Building CXX object CMakeFiles/ocl-stream.dir/src/ocl/OCLStream.cpp.o
[ 66%] Building CXX object CMakeFiles/ocl-stream.dir/src/main.cpp.o
/home/swchoi/src/BabelStream/src/main.cpp: In instantiation of ‘void run() [with T = float]’:
/home/swchoi/src/BabelStream/src/main.cpp:97:15:   required from here
/home/swchoi/src/BabelStream/src/main.cpp:280:42: error: ‘class Stream<float>’ has no member named ‘init_arrays’; did you mean ‘get_arrays’?
  280 |   auto initElapsedS = time([&] { stream->init_arrays(startA, startB, startC); });
      |                                  ~~~~~~~~^~~~~~~~~~~
      |                                  get_arrays
/home/swchoi/src/BabelStream/src/main.cpp: In instantiation of ‘void run() [with T = double]’:
/home/swchoi/src/BabelStream/src/main.cpp:99:16:   required from here
/home/swchoi/src/BabelStream/src/main.cpp:280:42: error: ‘class Stream<double>’ has no member named ‘init_arrays’; did you mean ‘get_arrays’?
  280 |   auto initElapsedS = time([&] { stream->init_arrays(startA, startB, startC); });
      |                                  ~~~~~~~~^~~~~~~~~~~
      |                                  get_arrays
gmake[2]: *** [CMakeFiles/ocl-stream.dir/build.make:90: CMakeFiles/ocl-stream.dir/src/main.cpp.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/ocl-stream.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
```

--- a/src/Stream.h
+++ b/src/Stream.h
@@ -41,6 +41,9 @@ class Stream
     virtual void nstream() = 0;
     virtual T dot() = 0;
 
+    // Reinitialize arrays to their starting values before a benchmark run.
+    virtual void init_arrays(T initA, T initB, T initC) = 0;
+
     // Set pointers to read from arrays
     virtual void get_arrays(T const*& a, T const*& b, T const*& c) = 0;
 };

